bolt-gcc-stage1.bash

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export TOPLEV=~/toolchain/gcc
+mkdir ${TOPLEV}
+cd ${TOPLEV}
+
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata/cc1
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata/cc1plus
+cd ${TOPLEV}/bolt-gcc
+GCCPATH=/usr/lib/gcc/x86_64-pc-linux-gnu/12.1.1
+GCCINSTPATH=${TOPLEV}/bolt-gcc
+BOLTPATH=~/toolchain/llvm/stage1/bin
+
+
+echo "Instrument clang with llvm-bolt"
+
+${BOLTPATH}/llvm-bolt \
+    --instrument \
+    --instrumentation-file-append-pid \
+    --instrumentation-file=${TOPLEV}/bolt-gcc/intrumentdata/cc1/cc1.fdata \
+    ${GCCPATH}/cc1 \
+    -o ${GCCINSTPATH}/cc1.inst
+
+echo "mooving instrumented binary"
+sudo mv ${GCCPATH}/cc1 ${GCCPATH}/cc1.org
```

Comment:
please dont sudo around like this :P

This kind of overwriting seems bad to me. Does the system compiler have to be destroyed like this?